managers/views.py

```diff
@@ -102,6 +102,12 @@ def get_success_url(self) -> str:  # pylint: disable=no-self-use
         return reverse("custom-projects-list")
 
     def form_valid(self, form: ProjectAdminForm) -> HttpRequest:
+        if self.request.user not in form.cleaned_data["managers"]:
+            assert not self.request.user.is_employee
```

Comment:
It seems overly cautious to me, since AFAIK we already successfully restrict access for unauthorised personel.
But, you know, personally I don't mind it much, really. 😁

Reply:
I gave assert here only for safety-check purposes. It already gave me feed back when I set there `assert self.request.user.is_manager` and I completely forgot about `ADMIN` role. So it might be helpful :)